### ISSUE TITLE
Show interactive help command to the user

### ISSUE THREAD

**Issue by mszostok** [kubeshop/botkube#683]
## Overview

Currently, when an invalid command is executed or when only `@BotKube` is typed, you get:

![Screen Shot 2022-08-09 at 12 44 52](https://user-images.githubusercontent.com/17568639/183629548-8b3cd71e-2988-441a-8649-f7e5dc7cdeee.png)

In the response, you get at least info that `/botkubehelp` will tell you more about possible commands. 

**Cons about such approach:**
- you need to type one more thing
- you get the response in a `plain text`. The best what you can do, is to copy-paste that command.
    ![Screen Shot 2022-08-09 at 12 46 49](https://user-images.githubusercontent.com/17568639/183629895-9c62cd14-fd84-46e5-8616-b0742726bacf.png)

## Desing

Slack design: https://www.figma.com/proto/iGGgUG9vn6C9WvDAFwKS9G/Botkube-Help---Slack?page-id=209%3A1337&node-id=269%3A1038&viewport=282%2C271%2C0.38&scaling=min-zoom&starting-point-node-id=269%3A1038

### Outdated design

![](https://user-images.githubusercontent.com/17568639/183634228-3014d5d3-183d-46f2-98bd-6d7343d61e6e.png)

## Acceptance criteria

- Help is shown when BotKube is installed and links to a Slack channel (first connection)
  - save state to not show it per app restart 
  - public message shown for everyone only once, right after installation for configured channels (initial config)
  - if the bot is invited to other channel, the message won't be printed
- A user runs `@BotKube help`
- The help dialog shows the dialog as per the design
- The buttons run the appropriate commands

## Out of scope

- A new user joins a BotKube channel (shown only to that user)
   - Reason: BotKube can be configured for multiple channels
- Notification presets configuration

## Notes

- Unknown: whether we can write E2E tests that check if the interactive buttons work
  - if Slack Go client doesn't have such method, skip that part - test the `@BotKube test` command only
- All messages are visible for everyone
- Assume #631 is resolved (merged into `main`)
- Initial focus on Slack
- Follow-up in future releases
- Ensure telemetry is in place 
- **it's ok that if you have multiple cluster on the same channel, then all return helper message**

**Comment by mszostok** [comment]
Related:
- https://github.com/kubeshop/botkube/issues/681

**Comment by mszostok** [comment]
If agreed, introduce the dedicated `@Botkube help` command as describe in https://github.com/kubeshop/botkube/issues/393

**Comment by mszostok** [comment]
## To discuss
- can we support all communication platforms?
- Can we test that e2e? It's it possible to "programmatically" click the button?

**Comment by mszostok** [comment]
PRs:
- https://github.com/kubeshop/botkube/pull/725
- https://github.com/kubeshop/botkube/pull/727
- https://github.com/kubeshop/botkube/pull/738
